ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -3434,6 +3434,8 @@ tests:
   cron: 43 6 30 * *
   steps:
     cluster_profile: vsphere-connected-2
+    env:
+      FOLDER: default
```

Comment:
Suggest to update the setting for `vsphere-upi-encrypt-f28-destructive` job too, also 4.18 ci jobs.